docs/standard/analyzers/platform-compat-analyzer.md

```diff
@@ -49,30 +49,39 @@ The platform compatibility analyzer is one of the Roslyn code quality analyzers.
 
 For more information, see [examples of how the attributes work and what diagnostics they cause](#examples-of-how-the-attributes-work-and-what-diagnostics-they-produce).
 
+## How the analyzer recognise TFM target plaforms
+
+The analyzer does not check TFM target platforms from MSBuild properties such as <`TargetFramework>` or `<TargetFrameworks>`, if the TFM has a target platform MSBuild would inject `SupportedOSPlatform` attribute with the targeted platform name in the `AssemblyInfo.cs` file which is consumed by the analyzer. For example if the TFM is `net5.0-windows`, MSBuild would inject `[assembly: System.Runtime.Versioning.SupportedOSPlatform("windows7.0")]` attribute into the `AssemblyInfo.cs` and the entire assembly would considered to be windows only. Therefore calling a windows only APIs versioned with 7.0 or below from windows targeted project would not cause any warning.
+
+  > [!NOTE]
+  > In case the `AssemblyInfo.cs` file generation is disabled for the project (i.e. `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>`) the required assembly level `SupportedOSPlatform` attribute could not be added by MSBuild, therefore you could see warnings for a platform specific APIs usage even if you are targeting that platform. In this case you might want to enable the `AssemblyInfo.cs` file generation or add the attribute manually in your project.
```

Comment:
This paragraph is a duplicate from the other article.

Not sure if it's okay to keep the duplication (with risk of them getting out-of-sync), or if this should link to the other article, or if it should be in an INCLUDE file.

Reply:
Yes, it is duplicate, the conceptual doc is a more detailed version of the rule doc, they do not have to be exactly same so it's OK, the pages have a link to each other